js/demo_policy/test_data/sessions/oci-manifest-lookup-failure.yml

```diff
@@ -0,0 +1,7 @@
+- type: Exchange
+  request: |
+    !OciManifest
+    image: registry.testing.lan/nonexistent-image:1.0.0
```

Comment:
Same here for the the image URI: `example.test/nonexistent-image:1.0.0` or similar.